pwm/pwm.sls

```diff
@@ -0,0 +1,110 @@
+#installs packages needed for PWM, downloads the WAR file and installs it into tomcat
+{%- from 'pwm/map.jinja' import data with context %}
```

Comment:
```suggestion
{%- from tpldir ~ '/map.jinja' import data with context %}
```

Reply:
Wait, where is `map.jinja`? It's not in the review?

Reply:
Previously had a map.jinja before I took it out and replaced it with pillar data. Deleting this line from the files.